domains/test/service/src/domain.rs

```diff
@@ -143,27 +145,38 @@ where
         role: Role,
         mock_consensus_node: &mut MockConsensusNode,
     ) -> Self {
-        let BootstrapResult {
-            domain_instance_data,
-            domain_created_at,
-            imported_block_notification_stream,
-        } = fetch_domain_bootstrap_info::<Block, _, _>(&*mock_consensus_node.client, domain_id)
-            .await
-            .expect("Failed to get domain instance data");
-        let chain_spec = create_domain_spec(domain_instance_data.raw_genesis);
-        let key_seed = Runtime::to_seed(key);
-        let domain_config = node_config(
+        let mut domain_config = node_config(
             domain_id,
             tokio_handle.clone(),
-            key_seed,
+            Runtime::to_seed(key),
             domain_nodes,
             domain_nodes_exclusive,
             role,
-            BasePath::new(base_path.path().join(format!("domain-{domain_id:?}"))),
-            Box::new(chain_spec) as Box<_>,
+            base_path.clone(),
+            Box::new(create_domain_spec()) as Box<_>,
         )
         .expect("could not generate domain node Configuration");
 
+        let domain_backend = sc_service::new_db_backend::<Block>(domain_config.db_config())
+            .expect("Failed to create domain backend: {error:?}");
```

Comment:
I’m not sure why the code is failing here on Linux. It might help to log more details here.

Also, expect already logs the error, and format strings require macros to work:
```suggestion
            .expect(&format!("Failed to create domain backend: {domain_id:?} {role:?} {base_path:?}"));
```